crates/rpc-types/src/eth/transaction/mod.rs

```diff
@@ -69,7 +69,9 @@ pub struct Transaction {
     #[serde(flatten, skip_serializing_if = "Option::is_none")]
     pub signature: Option<Signature>,
     /// The chain id of the transaction, if any.
+    #[serde(default)]
     #[serde(with = "alloy_serde::u64_hex_opt")]
+    #[serde(skip_serializing_if = "Option::is_none")]
```

Comment:
can we collapse them into a single serde attribute?
the skip is also fine because this field should be ignored